src/api/controllers/ActionUserController.js

```diff
@@ -1,17 +1,88 @@
-module.exports = {
-    addAttendant: (req, res) => {
+const Action = require("../models/ActionModel");
+const User = require("../models/UserModel");
+const mongo = require("mongoose");
+
+mongo.set("useFindAndModify", false);
+
+const updateAttendants = (req, res, add) => {
+  const userId = req.userData.userId;
+  const actionId = req.params.action_id;
+  var actionQuery = {};
+  var userQuery = {};
+
+  if (add) {
+    actionQuery["$addToSet"] = { attendees: userId };
+    userQuery["$addToSet"] = { actionsAttended: actionId };
+  } else {
+    actionQuery["$pull"] = { actionsAttended: actionId };
+    userQuery["$pull"] = { actionsAttended: actionId };
+  }
 
-    },
+  Action.findOneAndUpdate({ _id: actionId }, actionQuery)
+    .then((action) => {
+      if (action) {
+        User.findOneAndUpdate({ _id: userId }, userQuery)
+          .then(() => {
+            return res.status(201).send();
```

Comment:
I feel like **200** is more appropriate as we're not creating a document but rather updating it.
```suggestion
            return res.status(201).send();
```